kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/KmPrefSettings.java

```diff
@@ -0,0 +1,124 @@
+package com.applozic.mobicomkit.uiwidgets.kommunicate;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.text.TextUtils;
+
+import com.applozic.mobicomkit.api.MobiComKitClientService;
+import com.applozic.mobicommons.ApplozicService;
+
+public class KmPrefSettings {
+
+    private static KmPrefSettings kmPrefSettings;
+    private static SharedPreferences sharedPreferences;
+    private static final String ENABLE_FAQ_OPTION = "ENABLE_FAQ_OPTION";
+    private static final String ENABLE_SPEECH_TO_TEXT = "ENABLE_SPEECH_TO_TEXT";
+    private static final String ENABLE_TEXT_TO_SPEECH = "ENABLE_TEXT_TO_SPEECH";
+    private static final String SEND_MESSAGE_ON_SPEECH_END = "SEND_MESSAGE_ON_SPEECH_END";
+    private static final String SPEECH_TO_TEXT_LANGUAGE = "SPEECH_TO_TEXT_LANGUAGE";
+    private static final String TEXT_TO_SPEECH_LANGUAGE = "TEXT_TO_SPEECH_LANGUAGE";
+
+    private KmPrefSettings(Context context) {
+        if (!TextUtils.isEmpty(MobiComKitClientService.getApplicationKey(context))) {
+            sharedPreferences = context.getSharedPreferences(MobiComKitClientService.getApplicationKey(context), Context.MODE_PRIVATE);
```

Comment:
`sharedPreferences` we can check if we can create with some constant  name  and clear in logout

Reply:
Its already live with appId name. If we change it, it might break the existing application.

Reply:
okay